From 05b9f11251b58e6e124f729fd1bfa94052e526c6 Mon Sep 17 00:00:00 2001
From: Li Jun <jun.li@nxp.com>
Date: Fri, 8 Dec 2017 22:26:49 +0800
Subject: [PATCH 3092/5242] MLK-17128-3 staging: typec: reading max_snk_mw
 setting from OF

commit  070f35aae6881b2659e1ea4c879f8fe88eef4dc7 from
https://source.codeaurora.org/external/imx/linux-imx.git

It's missing while do typec related properties read, sink
needs it to check if the provided PDO can match the required
power.

Acked-by: Peter Chen <peter.chen@nxp.com>
Signed-off-by: Li Jun <jun.li@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/staging/typec/tcpci.c |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/staging/typec/tcpci.c b/drivers/staging/typec/tcpci.c
index 572d0ff..e0daf9a 100644
--- a/drivers/staging/typec/tcpci.c
+++ b/drivers/staging/typec/tcpci.c
@@ -745,6 +745,8 @@ static int tcpci_parse_config(struct tcpci *tcpci)
 						&tcfg->max_snk_mv) ||
 		device_property_read_u32(tcpci->dev, "max-snk-ma",
 						&tcfg->max_snk_ma) ||
+		device_property_read_u32(tcpci->dev, "max-snk-mw",
+						&tcfg->max_snk_mw) ||
 		device_property_read_u32(tcpci->dev, "op-snk-mw",
 						&tcfg->operating_snk_mw))
 		goto snk_setting_wrong;
-- 
1.7.9.5

